tools/check_reordering.py

```diff
@@ -18,7 +18,7 @@ def mapPathToSource(origName: Path) -> Path:
     # Try to map built path to the source path
     parts = origName.parts
     if parts[0] == "build":
-        parts = parts[1:]
+        parts = parts[2:]
```

Comment:
? . What is this change for?

Reply:
The output path of this script is meant to point to the C file, so you can ctrl-click it on vscode and open the corresponding file. It currently it has the version as prefix so you can't do that right now.

```
Symbol Name,Build Address,Build File,Expected Address,Expected File,Difference,GOOD/BAD/MISSING
D_80936CDA,80936AD0,n64-us/src/overlays/actors/ovl_En_Kusa/z_en_kusa.c,80936CDA,n64-us/src/overlays/actors/ovl_En_Kusa/z_en_kusa.c,-20A,BAD
D_80936CDC,80936AD2,n64-us/src/overlays/actors/ovl_En_Kusa/z_en_kusa.c,80936CDC,n64-us/src/overlays/actors/ovl_En_Kusa/z_en_kusa.c,-20A,BAD
```

Notice the `n64-us/src/overlays/actors/ovl_En_Kusa/z_en_kusa.c`

By changing this line the output doesn't have the version as prefix on the path anymore

```
Symbol Name,Build Address,Build File,Expected Address,Expected File,Difference,GOOD/BAD/MISSING
D_80936CDA,80936AD0,src/overlays/actors/ovl_En_Kusa/z_en_kusa.c,80936CDA,src/overlays/actors/ovl_En_Kusa/z_en_kusa.c,-20A,BAD
D_80936CDC,80936AD2,src/overlays/actors/ovl_En_Kusa/z_en_kusa.c,80936CDC,src/overlays/actors/ovl_En_Kusa/z_en_kusa.c,-20A,BAD
```

Reply:
Could you add a comment that it is stripping off build and version from the path. Perhaps something like `# Strip off build and version`?